contracts/staking/test/rewards_test.ts

```diff
@@ -602,6 +620,43 @@ blockchainTests.resets('Testing Rewards', env => {
                 membersRewardVaultBalance: rewardsForDelegator[1],
             });
         });
+        it('Should collect fees correctly when there is a payout for `currentEpochBalance` but not `nextEpochBalance`', async () => {
```

Comment:
Q: Is it even possible to do a payout on `nextEpochBalance`?

Reply:
Updated to clarify.